src/asciidocParser.ts

```diff
@@ -53,8 +50,18 @@ export class AsciidocParser {
           this.errorCollection.clear();
         }
 
-        const memoryLogger = asciidoctor.MemoryLogger.create();
-        asciidoctor.LoggerManager.setLogger(memoryLogger);
+        const memoryLogger = this.processor.MemoryLogger.create()
+        this.processor.LoggerManager.setLogger(memoryLogger)
+
+        this.registry = this.processor.Extensions.create()
+
+        // highlightjsAdapter.register(this.processor)
```

Comment:
I disabled this because we're still thinking through it and also because it wasn't working with the registry.

Reply:
> and also because it wasn't working with the registry.

The reason is that the highlight.js adapter is not an Asciidoctor extension but a custom syntax highlighter. So we need to pass the "asciidoctor instance" to be able to call `asciidoctor.SyntaxHighlighter.for('highlight.js')`.

The syntax highlighter registry and the extension registry are two things